src/runtimes/node/parser/exports.ts

```diff
@@ -39,6 +39,11 @@ export const traverseNodes = (nodes: Statement[], getAllBindings: BindingMethod)
     }
 
     const esmHandlerExports = getNamedESMExport(node, 'handler', getAllBindings)
+    const esmConfigExports = getNamedESMExport(node, 'config', getAllBindings)
```

Comment:
I'm trying to think of the implications of having both this and this: https://github.com/netlify/zip-it-and-ship-it/blob/c8f1b138495bbd4e102177f10523b8bdd6d71c3c/src/runtimes/node/parser/exports.ts#L79

I think it just means that if you do:

```js
export { config }
export const config = { ... }
```

... then the second one takes precedence?

Reply:
That would be an error. You can't have two exports with the same name.

Reply:
Right, we don't detect that here, so as far as the static analysis goes, the second config block would take precedence?

Reply:
It would have a runtime error though, so I'm not sure it matters what happens?